### ISSUE TITLE
Get aex9 transfers filtered by contractId and accountId

### ISSUE THREAD

**Issue by CedrikNikita** [aeternity/ae_mdw#1580]
Currently in the wallet we have an assets list that contain all the aex9 tokens user have balance on (`/v2/aex9/account-balances/${address}`). Not sure about all the ways across aeternity universe of getting such tokens on users account, but main source is of course DEX. And we are searching for the way to show transactions only related to a particular address and particular token, with as less requests as possible.

**Comment by jyeshe** [comment]
Hey @CedrikNikita this is a new endpoint, we are moving things to `/v3` (there will be a new yaml for it, sorry for the inconvenience). 

You can use either `sender` or `recipient` param like 
https://testnet.aeternity.io/mdw/v3/aex9/ct_5gdzRK8XkNLY73LSR8BNVqZ6cKMC9f7hKG1cxyXhM11CkGvt8/transfers?sender=ak_2MvFv6inmUCS5XDaxerpVgwmtHugvYC1oMKQdSw1qeMuhcTPUY
or
https://testnet.aeternity.io/mdw/v3/aex9/ct_5gdzRK8XkNLY73LSR8BNVqZ6cKMC9f7hKG1cxyXhM11CkGvt8/transfers?recipient=ak_y87WkN4C4QevzjTuEYHg6XLqiWx3rjfYDFLBmZiqiro5mkRag

**Comment by CedrikNikita** [comment]
This two parameters making it harder to work with it. Because we are trying to show the user all the transactions related to a particular `contractId`. Now with two different parameters we would need to:
- make two requests in parallel instead of one
- merge this list together and sort them on our side
- remember both of the `next`s and trying to make a logic which list of transaction is over (and we need to load more) on scrolling and which is not

**Comment by peronczyk** [comment]
Hi. Currently the @aeternity/superhero team is working on handling not only AE protocol, but also **Bitcoin** and **Ethereum**. This requires us to avoid any hacks and protocol specific workarounds so the wallet is as plugabble as it can be.

As @CedrikNikita mentioned in his comment: having two separate endpoints for `sender` and `receiver` is not usable for us. We are not able to keep two separate next page urls when trying to obtain one list. It would be possible if our wallet would be focused on AE protocol only, but we need to keep a lot of data and splitting the calls makes our work insanely hard. Please see for example how Etherscan is handling this: https://docs.etherscan.io/api-endpoints/accounts#get-a-list-of-erc20-token-transfer-events-by-address

So maybe there is a possibility of having **one endpoint for fetching all AEX9 token transactions**?

@jyeshe and @sborrazas @aeternity/core

**Comment by peronczyk** [comment]
> This two parameters making it harder to work with it. Because we are trying to show the user all the transactions related to a particular `contractId`. Now with two different parameters we would need to:
> 
> * make two requests in parallel instead of one
> * merge this list together and sort them on our side
> * remember both of the `next`s and trying to make a logic which list of transaction is over (and we need to load more) on scrolling and which is not
> 
> Also without the `decimals` field we will not know how to present this `amount` field to the user.

I created separate issue for this: https://github.com/aeternity/ae_mdw/issues/1678
